src/data_tools/src/benchmark.cpp

```diff
@@ -498,8 +664,8 @@ mbes_ping::PingsT registration_summary_benchmark::get_submap_pings_index(const m
     return pings_i;
 }
 
-pair<double, cv::Mat> track_error_benchmark::compute_draw_error_consistency_map(mbes_ping::PingsT& pings)
-{
+vector<vector<vector<Eigen::MatrixXd>>> track_error_benchmark::create_grids_from_pings(mbes_ping::PingsT& pings){
```

Comment:
">>>" will not work with some c++ compilers, there needs to be spaces in between.